lib/rules/no-console.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         docs: {
             description: "disallow the use of `console`",
             category: "Possible Errors",
-            recommended: true,
+            recommended: false,
```

Comment:
this isn’t a breaking change; could it land separately?

Reply:
I think it is a breaking change; projects that want to disallow `console` can currently rely on `eslint:recommended` to do so. By removing the rule we would no longer be providing that guarantee, and those projects would need to be told that they should migrate by configuring `no-console` explicitly.

We do sometimes loosen the default options for particular rules in a minor release, but in those cases I would say the impact of a false negative is usually lower.